bigquery/indexer.py

```diff
@@ -82,17 +82,14 @@ def _get_nested_mappings(schema, prefix=None):
     return nested if nested else None
 
 
-def _create_nested_mappings(es, index_name, table_name, sample_id_col,
-                            billing_project_id):
+def _create_nested_mappings(es, index_name, table, sample_id_col):
```

Comment:
Remove pandas import at top. Though I think we still need pandas in requirements-to-freeze.txt -- can you check? I think you can remove pandas_gbq from requirements-to-freeze.txt , can you try that also?

Reply:
You're right, we still needs pands. But removed pandas-gbq.